Code2Gether-Discord-Bot.Library/BusinessLogic/PingLogic.cs

```diff
@@ -19,7 +20,7 @@ public PingLogic(ILogger logger, ICommandContext context, int latency)
             _latency = latency;
         }
 
-        public Embed Execute()
+        public Task<Embed> Execute()
         {
             _logger.Log(_context);
 
```

Comment:
`await _context.Guild.GetCurrentUserAsync()`
then get results' properties that you need.

Reply:
`await context.Channel.SendMessageAsync(...)`

Reply:
`await temporaryMessage.DeleteAsync()`